gpii/node_modules/ontologyHandler/test/node/OntologyHandlerTests.js

```diff
@@ -79,6 +79,159 @@ https://github.com/gpii/universal/LICENSE.txt
         ]
     };
 
+    var floydWarshallTestsLength = 18;
+    var floydWarshallTests = [
+        {
+            name: "Initial 4 vertice, 3 edges",
```

Comment:
You ARE completely crazy - please read this review comment again. "vertices" *is* the accepted plural of vertex, what I have been commenting on since the beginning is your insistence that there is a word such as "vertice" : P

Please ensure that throughout

i) The singular word is vertex
ii) The plural word is vertices